src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -87,7 +88,11 @@ public class Entity implements Viewable, Tickable, Schedulable, Snapshotable, Ev
             EntityType.FISHING_BOBBER, EntityType.SNOWBALL, EntityType.EGG, EntityType.ENDER_PEARL, EntityType.POTION,
             EntityType.EYE_OF_ENDER, EntityType.DRAGON_FIREBALL, EntityType.FIREBALL, EntityType.SMALL_FIREBALL,
             EntityType.TNT);
-
+    private static final Set<EntityType> ALLOW_BLOCK_PLACEMENT_ENTITIES = Set.of(EntityType.ARROW, EntityType.ITEM,
```

Comment:
Do these come from the vanilla client?

Reply:
No they don't, I just added the ones that I thought of. Probably missing some